include/kernels/reduce_core.cuh

```diff
@@ -3,60 +3,68 @@
 #include <color_spinor_field_order.h>
 #include <blas_helper.cuh>
 #include <reduce_helper.h>
-#include <fast_intdiv.h>
+#include <reduction_kernel.h>
 
 namespace quda
 {
 
   namespace blas
   {
 
-    template <typename store_t, int N, typename y_store_t, int Ny, typename Reducer_>
+    template <typename real_, int n_, typename store_t, int N, typename y_store_t, int Ny, typename Reducer_>
```

Comment:
doxygen (should be easy to copy fom `multi_reduce_core`, otherwise ignore request for now)

Reply:
172f419b3